fastlane/lib/fastlane/actions/spm.rb

```diff
@@ -10,6 +10,15 @@ def self.run(params)
         cmd << "--configuration #{params[:configuration]}" if params[:configuration]
         cmd << "--disable-sandbox" if params[:disable_sandbox]
         cmd << "--verbose" if params[:verbose]
+        if params[:simulator]
+          simulator_flags = [
+            "-Xswiftc", "-sdk",
+            "-Xswiftc", "$(xcrun --sdk #{params[:simulator]} --show-sdk-path)",
+            "-Xswiftc", "-target",
+            "-Xswiftc", "x86_64-apple-ios$(xcrun --sdk #{params[:simulator]} --show-sdk-version | cut -d '.' -f 1)-simulator"
```

Comment:
Won't this build only for x86 arch? What about arm64? 👀

Reply:
Good point, however I don't have anything in mind now. What do you suggest? building the package twice once for each simulator ? or adding an option that user can also define the arch?

Reply:
I'd probably allow the user to decide their own adventure, but at the same time provide sensible defaults, which IMO should be arm64 at this point. Perhaps `x86_64`, `arm64` and `both` could be the options? Or maybe not `both` and if they want they can invoke the action twice? 

Open to other thoughts :)

Reply:
instead of adding `both` make it that the target option is a list of comma separated values, and iterate over the `split(',')` of it?

Reply:
I was leaning towards not allowing multiple archs to be built at once, but this sounds better ☝️

Reply:
Good point guys. Thanks. I will try to implement this idea.

Reply:
Hey guys, Please feel free to take a look for new changes adding support to simulator. I was taking a look at swift build script here [github](https://github.com/apple/swift/blob/main/utils/build-script-impl) and found that the currect architectures are `arm64`, `arm64e` and `x86_64` which the first one is only for `macosx` therefor I didn't add it and only the other two.
Also I wasn't sure how to manage the support of multiple archs at once, so for now I added support only one arch at a time.
I'm open to any suggestion/ comment/ changes.